test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/Roundtrip/ContextUrlWriterReaderTests.cs

```diff
@@ -921,8 +926,8 @@ public void ExpandedEntity()
         // V4 Protocol Spec Chapter 10.10: Projected Expanded Entities (not contain select)
         // Sample Request: http://host/service/Employees(0)?$expand=AssociatedCompany
         [Theory]
-        [InlineData(ODataVersion.V4, "\"{0}$metadata#Employees(AssociatedCompany)/$entity\"")]
-        [InlineData(ODataVersion.V401, "\"{0}$metadata#Employees(AssociatedCompany())/$entity\"")]
+        [InlineData(ODataVersion.V4, "\"{0}$metadata#Employees(AssociatedCompany,AssociatedCompany())/$entity\"")]
+        [InlineData(ODataVersion.V401, "\"{0}$metadata#Employees(AssociatedCompany,AssociatedCompany())/$entity\"")]
         public void ExpandedMultiSegmentEntity(ODataVersion version, string contextString)

```

Comment:
Here again, we should just have a single string defined for the contextString and run the test for each ODataVersion using that same string definition (to enforce that the two versions don't diverge)

Reply:
Consolidated the two test cases. Thanks!
I have also checked the rest of the changes, looks like there are no other cases similar to these that can be combined.

---
In reply to: [226131792](https://github.com/OData/odata.net/pull/1286#discussion_r226131792) [](ancestors = 226131792)